### ISSUE TITLE
Getting "Operation Failed" when trying to save images

### ISSUE THREAD

**Issue by LordManhattan** [ccrama/Slide#1001]
Video: https://vid.me/DqBn

No idea why it's not working, but it has always been like this. I just got reminded about it again now when I wanted to save an image. I checked the logcat, but I can't find any errors, so I have no idea what's going on. 

Slide version: 4.6.0-42-8d2e23c-dirty

I use the stock Sony Album app, which should open the image after I've saved it, but then I get the "operation failed" error, and gets kicked back to Slide. No signs after the image(s) in the gallery or in any folders.

**Comment by ccrama** [comment]
That doesn't have to do with Slide, it's an issue with either your file system or your gallery app. Slide only shows the notification if the file was saved and it can confirm the file exists

**Comment by LordManhattan** [comment]
But how can it not be Slide when it only happens when using Slide? It has never happened before. Your explanation is logical though, but I just find it funny how it's only Slide that is able to trigger the issue.

**Comment by ccrama** [comment]
Any app that uses the android media library probably has that issue. I'll try to find an app that does and we will see!

**Comment by LordManhattan** [comment]
But doesn't most or all galleries, music player, or to put it simply - all media libraries, Android media library? I still only experience this issue with Slide, and don't have any issues with other media apps or other apps that are able to save media. 

I'm running stock 5.1.1 (Sony), I'm rooted, but I do not use Xposed or anything like that. I use the stock Sony Album app, but I haven't had any issues with it in the past. 

I can't download some 3rd party gallery apps, try to save some images from Slide and open them in the 3rd party galleries. I'll see if I'm able to catch it in the logcat while doing so.

**Comment by LordManhattan** [comment]
I managed to catch it: 

`"02-16 17:30:53.724 E/SpfParser(19605): File not found"`

When I choose to open the image in Sony Album:

`"02-16 17:37:14.303 E/GoogleTagManager(6749): Invalid macro: _gtm.loadEventEnabled"`

I installed "Piktures" and "Foto Gallery" from the Play Store to see if those were able to locate and open the image, and they do, both of them. Sony Album is not, so it's pretty obvious now that it's not an issue with Slide, which you've already told me, but it's nice to have it confirmed.

**Comment by ccrama** [comment]
The sync part is irrelevant to me as he probably uses some other method of saving images. I'm using the code provided by Google, so will see if they have a fix for the issue.

**Comment by ccrama** [comment]
Should be fixed now, can you confirm? I'll re-open it if it doesn't work

**Comment by LordManhattan** [comment]
I'm afraid the issue is still there. I checked the Logcat if there's anything unusual there, but I can't find anything(?)

https://drive.google.com/file/d/0B-7kjd-IE_zwbm8wWUxsUkRWNGs/view?usp=docslist_api

And just to confirm, the latest app version is 4.6.2-18-7b1066d?

**Comment by ccrama** [comment]
What is your default image folder? If it's still failing, that means you are missing a photos folder. And yeah, should be the right version

**Comment by LordManhattan** [comment]
/storage/emulated/0/Pictures

Apps have their own image folder here. Slide is missing a folder. Should it have one?

**Comment by ccrama** [comment]
No, it saves to "/sdcard/Pictures", which is the same as "/storage/emulated/0/Pictures" afaik. I am thinking it's device specific, because I haven't seen anyone else report this issue in a while (since v4.6). I'll re-open it just in case, though

**Comment by LordManhattan** [comment]
Okay, so I downloaded some images using Slide. Yes, I get Operation Failed when I try to open the images, but I went to /storage/emulated/0/Pictures to check if the images are there, and sure enough, they are. They don't want to be opened in the Sony Album app though, so... weird?! 

Other gallery apps open images from Slide just fine, but Sony Album also opens downloaded images from “all" the other Reddit apps that I've tried (Relay, Sync, Boost, Karma), so I have no idea what's going on here.
